scripts/build-and-publish-wit.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# This script builds and publishes the spin:up WIT package
+# NOTE: The package name and version are inferred from the
```

Comment:
Not any more _grin_

Reply:
Wait, no. This is still accurate.

Reply:
![](https://media1.tenor.com/m/De285fAXTuQAAAAC/futurama-technically-correct.gif)